kano_profile/badges.py

```diff
@@ -390,6 +390,31 @@ def save_app_state_variable_with_dialog(app_name, variable, value):
     save_app_state_with_dialog(app_name, data)
 
 
+def update_upwards_with_dialog(app_name, variable, value):
```

Comment:
Keeping the convention for now. At the future we can refactor this whole module to convert the `dialog` into a kwarg instead of reimplementing each of the fns from the apps.py module